src/lib/random.js

```diff
@@ -340,6 +341,88 @@ var $builtinmodule = function (name) {
         return new Sk.builtin.float_(c);
     });
 
+    mod.triangular = new Sk.builtin.func(function (low, high, mode) {
+        Sk.builtin.pyCheckArgs("triangular", arguments, 2, 3);
+
+        var rnd, sample, swap;
+
+        low = Sk.builtin.asnum$(low);
+        high = Sk.builtin.asnum$(high);
+        if (low > high) {
+            swap = low;
+            low = high;
+            high = swap;
+        }
+        if (mode === undefined) {
```

Comment:
What about passing `None` or any non numeric types here? We would need to check for those and throw an appropriate `TypeError` or `ValueError`.